README.md

```diff
@@ -70,22 +91,22 @@ Also, you can configure the product summary that is defined on minicart. See [he
 
 This app provides some CSS classes as an API for style customization.
 
-Class name | Description | Component Source
----|---|---
-`container` | The main container of minicart | [index](/react/index.js)
-`label` | Minicart icon label | [index](/react/index.js)
-`badge` | Minicart badge with the product quantity on it | [index](/react/index.js)
-`arrowUp` | Popup box arrow | [Popup](/react/components/Popup.js)
-`box` | The main container of the popup | [Popup](/react/components/Popup.js)
-`sidebarHeader` | Minicart sidebar header container | [Sidebar](/react/components/Sidebar.js)
-`sidebar` | Minicart sidebar main container | [Sidebar](/react/components/Sidebar.js)
-`sidebarOpen` | Active when the sidebar is opened | [Sidebar](/react/components/Sidebar.js)
-`content` | The container for the Minicart contents | [MinicartContent](/react/components/MinicartContent.js)
-`contentSmall` | The container for the minicart contents when on desktop size | [MinicartContent](/react/components/MinicartContent.js)
-`contentLarge` | The container for the minicart contents when on mobile size | [MinicartContent](/react/components/MinicartContent.js)
+Class name        | Description                    | Component Source
+----------------- | ------------------------------ | ------------------------
```

Comment:
(I think "indenting" this just makes it more difficult to add or remove an item of the table, because you always have to align with the wider column.)

Reply:
Yeah, I agree. Is just my TOC. :developers: